uninstall.php

```diff
@@ -11,4 +11,34 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() ) {
+	delete_data_multisite();
+} else {
+	delete_data();
+}
+
+/**
+ * Delete data for the current blog.
+ */
```

Comment:
Use proper documentation for the function.
```
/**
 * Delete plugin option data for the current blog.
 *
 * @since n.e.x.t
 */
```

Reply:
Thank you, added